sapp/ui/server.py

```diff
@@ -99,5 +99,12 @@ def start_server(
     if static_resources:
         application.static_folder = static_resources
     if debug:
-        CORS(application, resources={r"/graphql": {"origins": "http://localhost:3000"}})
+        default_port = int(os.environ.get("PORT"), 10) or 3000
+        host = os.environ.get("HOST") or "localhost"
+        CORS(
+            application,
+            resources={
+                r"/graphql": dict(origins="http://" + host + ":" + str(default_port))
```

Comment:
I think we prefers f-strings or `str.format` over concatenation

```suggestion
                r"/graphql": dict(origins="http://{hostname}:{port}".format(hostname=host, port=default_port)
```